store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10560,4 +10560,14 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Feature to enable delivery status notification</desc>
 </attr>
+
+<attr id="4136" name="zimbraFeatureExternalEmailWarningEnabled" type="boolean" cardinality="single" optionalIn="cos,account" flags="accountInherited,accountInfo" since="10.1.7">
```

Comment:
don't we need this at domain level?

Reply:
Do you want like optionalIn="cos,domain,account"? currently there is a backend bug, where account is only inheriting from cos not from domain for all such ldap attributes. https://synacor.atlassian.net/browse/ZCS-16534 and https://synacor.atlassian.net/browse/ZCS-16548
As per @dawoodshaikh on feature requirement, it was discussed to be at cos, account. Let discuss with @dawoodshaikh  if there is any specific scenarios which it need should be domain based

Reply:
@silentsakky  Setting it at global,domain level